src/main/kotlin/net/irisshaders/lilybot/utils/_Utils.kt

```diff
@@ -9,58 +10,26 @@ import dev.kord.core.kordLogger
 import kotlinx.coroutines.flow.toList
 
 /**
- * This is a simple function to get a value from the configuration database in an [EphemeralSlashCommandContext],
- * null check it, and respond accordingly. It takes a string input of the column that is requested and gets it, if it's
- * null a private error message (i.e. for commands staff run), and null is returned. If null this should be handled with
- * an elvis operator to return the action.
+ * This is a check to verify that no element of the guild config is null, since these are all non-nullable values, if
+ * any one of them is null, we fail with the unable to access config error message.
  *
- * @param inputColumn The column you wish to get from the database
- * @return The column value from the database or null
  * @author NoComment1105
  */
-suspend fun EphemeralSlashCommandContext<*>.getConfigPrivateResponse(inputColumn: String) =
-	DatabaseHelper.getConfig(guild!!.id, inputColumn) ?: run {
-		respond {
-			content = "**Error:** Unable to access config for this guild! Is your configuration set?"
-		}
-		null
-}
+suspend fun CheckContext<*>.configPresent() {
+	if (!passed) {
+		return
+	}
 
+	if (guildFor(event) == null) fail("Must be in a server")
```

Comment:
Does this text get included in a response?

Reply:
Yes